src/vonage/account.py

```diff
@@ -8,30 +12,38 @@ def get_balance(self):
     def topup(self, params=None, **kwargs):
         return self._client.post(self._client.host(), "/account/top-up", params or kwargs)
 
-    def get_country_pricing(self, country_code):
+    def get_country_pricing(self, country_code: str, type: str = 'sms'):
+        self._check_allowed_pricing_type(type)
+        return self._client.get(
+            self._client.host(), f"/account/get-pricing/outbound/{type}", {"country": country_code}
+        )
+
+    def get_all_countries_pricing(self, type: str = 'sms'):
+        self._check_allowed_pricing_type(type)
         return self._client.get(
-            self._client.host(), "/account/get-pricing/outbound", {"country": country_code}
+            self._client.host(), f"/account/get-full-pricing/outbound/{type}"
         )
 
-    def get_prefix_pricing(self, prefix):
+    def get_prefix_pricing(self, prefix: str, type: str = 'sms'):
+        self._check_allowed_pricing_type(type)
         return self._client.get(
-            self._client.host(), "/account/get-prefix-pricing/outbound", {"prefix": prefix}
+            self._client.host(), f"/account/get-prefix-pricing/outbound/{type}", {"prefix": prefix}
         )
 
-    def get_sms_pricing(self, number):
+    def get_sms_pricing(self, number: str):
         return self._client.get(
             self._client.host(), "/account/get-phone-pricing/outbound/sms", {"phone": number}
         )
 
-    def get_voice_pricing(self, number):
+    def get_voice_pricing(self, number: str):
         return self._client.get(
             self._client.host(), "/account/get-phone-pricing/outbound/voice", {"phone": number}
         )
 
     def update_default_sms_webhook(self, params=None, **kwargs):
         return self._client.post(self._client.host(), "/account/settings", params or kwargs)
 
-    def list_secrets(self, api_key):
+    def get_all_secrets(self, api_key):
```

Comment:
`list_secrets` would be the proper verbiage for this https://github.com/Nexmo/server-sdk-specification/blob/main/SPECIFICATION.md#verbs since we are showing multiple results but can't filter it.

Reply:
Will put this back, I see your point

Reply:
Have renamed